### ISSUE TITLE
Stop writing images directly to the `.minecraft` folder

### ISSUE THREAD

**Issue by NatoBoram** [MamiyaOtaru/voxelmap#60]
I think it's disrespectful to invade the user's `.minecraft` folder, especially since the `.minecraft/config` folder *and* the `.minecraft/voxelmap` are already used by VoxelMap. There are many, many mobs now that are all shown in this folder for no good reason. It would be appropriate to put them in their own folder, either `.minecraft/config/VoxelMap` or `.minecraft/voxelmap`.

**Comment by fooeyround** [comment]
I agree its a big clutter, also is the code here not complete or is it just weirdly written and without the build files?

**Comment by 0xfeeddeadbeef** [comment]
The code looks "weirdly written" because it is a decompiled source code of a discontinued mod named "Zan's Minimap".

**Comment by fooeyround** [comment]
Hmm but where is the voxelmap code that is in the mod? Is it not open or?

**Comment by NatoBoram** [comment]
It's closed source, just like Zan's Minimap. It's also probable that [fantahund99's fork](https://discord.gg/4nGrfZxb2a) will be closed source.

**Comment by fooeyround** [comment]
Wait then how do they get the source to fork? Do they tediously just bring it from.intermediary to yarn and remake build scripts?
